### PR TITLE
add simple MIT license

### PR DESCRIPTION
**Pull request by colearendt**
I'm not sure whether I should clarify anywhere in `LICENSE.md` what this license applies to? It is in the _repository_ and it says "this software," so I think it should inherently apply to the code / docs in the repository and nothing else, correct?

For reference: [licensing images themselves is complex](
https://www.linuxfoundation.org/resources/publications/docker-containers-what-are-the-open-source-licensing-considerations/) because it includes the licenses of every piece of software included in _any layer_ in the image.

### CODE REVIEW

File: server-pro/Dockerfile [hunk 1]
```diff
@@ -123,7 +123,8 @@ RUN apt-get update --fix-missing \
     && gdebi --non-interactive ${RSP_NAME}-${RSP_VERSION}-amd64.deb \
     && rm ${RSP_NAME}-${RSP_VERSION}-amd64.deb \
     && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -rf /var/lib/rstudio-server/r-versions
```
Comment (colearendt) [inline server-pro/Dockerfile:127]: Woops - this was supposed to go in a different PR 🙈

**Review by akgold — APPROVED** [review]
My comment from the other repo here too. I think it's known that "this software" refers to the code actually in the repo.

Good with me, assuming you're removing that code before merging.
